wmt-etl/README.md

```diff
@@ -0,0 +1,39 @@
+# WMT Alpha Data Integration Prototype
+
+This section of the prototype illustrates how we will implement data integration.
+
+[Pandas](http://pandas.pydata.org/) is a flexible and powerful Python library for data manipulation and analysis.
+
+It provides rich functionality for working with database, file based and API abased data souces and for converting between them.
+
+### Set-up Python Dev environment
+Python development uses [virtualenvwrapper](http://virtualenvwrapper.readthedocs.io/en/latest/index.html) to provide dependency isolation.
+
+To set up Python and VirtualEnv install as follows:
+
+`brew install python`
+`pip install virtualenvwrapper`
+
+Add the following lines to your `~/.bash_profile`:
+
+```
+export WORKON_HOME=$HOME/.virtualenvs
+export PROJECT_HOME=$HOME/Devel
+source /usr/local/bin/virtualenvwrapper.sh
+```
+
```

Comment:
Is it worth mentioning that a `source ~/.bash_profile` will be required before these environment variables are set?